index.html

```diff
@@ -1256,31 +1346,80 @@
   // handle gUM error here
 });
 
-// ... later, in the rendering loop ...
+var gl = canvas.getContext("webgl2");
+// Activate the extension to use single component R32F texture format.
+gl.getExtension('EXT_color_buffer_float');
+
+// Later, in the rendering loop ...
```

Comment:
Could be have this be in another box in the spec, so it is super clear that this is a separate part of the code